content/ja/news.md

```diff
@@ -1,35 +1,49 @@
 ---
 title: ニュース
 sidebar: false
-newsHeader: "numpy.orgが日本語とポルトガル語に対応しました。"
-date: 2023-08-02
-newsLink: /ja/news
+newsHeader: "NumPy 1.26.0 がリリースされました。"
+date: 2023-09-16
 ---
 
-### numpy.orgが日本語とポルトガル語に対応しました。
+### NumPy 1.26.0 がリリースされました。
 
-_2023年8月2日_ -- numpy.orgが2つの言語で利用可能になりました：
-日本語とポルトガル語。 私たちの熱心なボランティアがいなければ、これは不可能です：
+_2023年9月16日_ -- [NumPy 1.26.0](https://numpy.org/doc/stable/release/1.26.0-notes.html)がリリースされました。 今回のリリースの目玉機能は次のとおりです。
 
-_ポルトガル語:_
-* Melissa Weber Mendonça (melissawm)
+* Python 3.12.0 のサポート
+* Cython 3.0.0 への互換性
+* Mesonビルドシステムの利用
+* SIMD サポートの改善
+* f2py のバグ修正, meson と bind(x) のサポート
+* 更新された BLAS/LAPACK の高速化ライブラリのサポート
+
+Numpy 1.26.0 は 1.25 からの互換性を保持しています。Mesonビルドシステムへの移行とCython 3.0.0へのサポートが目的のリリースです。 合計20人がこのリリースに貢献し、59個のプルリクエストがマージされました。
+
+このリリースでサポートされている Python のバージョンは3.9から 3.12 です。
+
+### numpy.orgが日本語とポルトガル語で利用可能になりました
+
+_2023年4月2日_ -- numpy.orgが2つの言語で利用可能になりました： 日本語とポルトガル語。 熱心なボランティアがいなければ、このプロジェクトは不可能でした：
+
+_ポルトガル語_
+* Melissa Weber Mendonça (melissawm)
 * Ricardo Prins (ricardoprins)
-* Getúlio Silva (getuliosilva)
+* Getúlio Silva (getuliosilva)
+>>>>>>> 4528610 (New translations news.md (Japanese))
```

Comment:
This looks like it should be removed.

Reply:
Thanks. Not sure how I missed these merge artifacts.